tests/ember_debug/object-inspector-test.js

```diff
@@ -659,15 +660,28 @@ module('Ember Debug - Object Inspector', function (hooks) {
       3,
       'Correctly merges properties'
     );
-    assert.equal(message.details[3].properties[0].name, 'toString');
-    assert.equal(message.details[3].properties[1].name, 'hasChildren');
+
+    const toString = A(message.details[3].properties).findBy(
+      'name',
+      'toString'
+    );
+    const hasChildren = A(message.details[3].properties).findBy(
+      'name',
+      'hasChildren'
+    );
+    const expensiveProperty = A(message.details[3].properties).findBy(
+      'name',
+      'expensiveProperty'
+    );
+    assert.ok(toString);
+    assert.ok(hasChildren);
```

Comment:
I know you didn't introduce this here (it was also wrong in the original version) but IMHO if you have more than 1 assertion in a test, all of the assertions **must** have a message (otherwise tracking down which one is failing is really annoying).

```suggestion
    assert.ok(toString, 'has toString');
    assert.ok(hasChildren, 'has hasChildren');
```